Formula/piknik.rb

```diff
@@ -0,0 +1,55 @@
+class Piknik < Formula
+  desc "Copy/paste anything over the network"
+  homepage "https://github.com/jedisct1/piknik"
+  url "https://github.com/jedisct1/piknik/archive/0.9.tar.gz"
+  sha256 "4b61e78d7c2f4ddc69cffe9e800a6ca03af8e90540a089469def615388087983"
+  head "https://github.com/jedisct1/piknik.git"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/jedisct1/"
+    dir.install Dir["*"]
+    ln_s buildpath/"src", dir
+    cd dir do
+      system "godep", "restore"
+      system "go", "build", "-o", bin/"piknik", "."
+      (prefix/"etc/profile.d").install "zsh.aliases" => "piknik.sh"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    In order to get convenient shell aliases, put something like this in #{shell_profile}:
+      source $(brew --prefix)/etc/profile.d/piknik.sh
```

Comment:
Need to quote command substitution. `source "$(brew --prefix)/etc/profile.d/piknik.sh"`.